guides/micronaut-rest-assured/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Using Rest-Assured in Micronaut application",
```

Comment:
"in Micronaut application" should be with "in a Micronaut application" or "in Micronaut applications"